### ISSUE TITLE
Prepare on awake setting for the recorder still causes stutter

### ISSUE THREAD

**Issue by ctrlaltdel85** [natmlx/videokit#20]

**Comment by olokobayusuf** [comment]
@ctrlaltdel85 macOS only? Or also on device?

**Comment by ctrlaltdel85** [comment]
Both, I also see a single black frame flicker at the start even in followup recordings.

**Comment by olokobayusuf** [comment]
Hey @ctrlaltdel85 sorry for the late follow-up on this. The black frame you mention sounds like a separate issue. VideoKit enforces the very first frame in a recording to be a video frame, so as to specifically prevent a black frame from being the starting video frame. With this in mind, it could be that the frame you are committing is black. What's your recording setup?

The stutter issue has been suppressed (seems like there's still a very tiny stutter, but it's barely noticeable). The next VideoKit update is ready; it's just waiting on a NatML update that should be ready later today or tomorrow.

**Comment by olokobayusuf** [comment]
This has been addressed in [VideoKit 0.0.12](https://github.com/natmlx/videokit/releases/tag/0.0.12).
